### DISCUSSION TITLE
Is it possible to configure multiple rules in the same yaml rule file ?

### DISCUSSION THREAD

**Discussion by alexku7** [jertel/elastalert2#1117] [Q&A] [answered]
Probably I am missing something but I can't add another rule to the same yaml file. 
It takes the latest rule in the list.

I tried
name: rule1
.
.
.

name: rule2

Also tried diffrent combination like

- name: rule1
- name: rule1
What am I missing here ?

**Comment by jertel** [comment, marked as answer]
No, each rule must be defined in its own yaml file.
